ydb/_grpc/grpcwrapper/ydb_topic.py

```diff
@@ -69,6 +69,13 @@ def from_proto(msg: Optional[ydb_topic_pb2.SupportedCodecs]) -> "SupportedCodecs
     def to_public(self) -> List[ydb_topic_public_types.PublicCodec]:
         return list(map(Codec.to_public, self.codecs))
 
+    @staticmethod
+    def from_public(codecs: Optional[List[ydb_topic_public_types.PublicCodec]]) -> "SupportedCodecs":
```

Comment:
1. Optional for result
2. Supported codec require Codec instead of PublicCodec
3. from_public need to accept list of PublicCodec or int